app/assets/stylesheets/base/_functions.scss

```diff
@@ -0,0 +1,7 @@
+@function theme-light($color-key) {
+	@return map-get($colors-light, $color-key);
+}
+
+@function theme-dark($color-key) {
+	@return map-get($colors-dark, $color-key);
+}
```

Comment:
did u split these out of `_mixins.scss` because they're functions and not mixins?

that's semantically correct but im not sure how useful it is. 
i guess maybe down the road it might be but i don't see the need for it rn.

Reply:
yeah, I'll move them back

Reply:
yeah, I'll move them to mixins